### PR TITLE
GH-28 Added javax.activation to default xjcLibts

### PR DESCRIPTION
**Pull request by markarasev**
Solves GH-28

Works for both openjdk 11.0.7 and 1.8.0_252.

### CODE REVIEW

**Comment by tOverney** [comment]
Ran into the same issue, and this would indeed do the trick!

**Comment by benwaffle** [comment]
@benmccann can we get this merged?

File: src/main/scala/com/github/retronym/sbtxjc/SbtXjcPlugin.scala [hunk 1]
```diff
@@ -39,7 +39,8 @@ object SbtXjcPlugin extends AutoPlugin {
     xjcPlugins        := Seq(),
     xjcLibs           := Seq(
       "org.glassfish.jaxb" % "jaxb-xjc"  % "2.2.11",
-      "com.sun.xml.bind"   % "jaxb-impl" % "2.2.11"
+      "com.sun.xml.bind"   % "jaxb-impl" % "2.2.11",
+      "javax.activation" % "activation" % "1.1.1"
```
Comment (benmccann) [inline src/main/scala/com/github/retronym/sbtxjc/SbtXjcPlugin.scala:43]:
```suggestion
      "javax.activation"   % "activation" % "1.1.1"
```

**Comment by tOverney** [comment]
@benmccann could we get this version published please :)

**Comment by benmccann** [comment]
I have not used sbt in the past 5 years, so am not the right person to ask. You should ask someone from the sbt team instead

**Pull request merged**